<launch>
  <!--启动的节点-->
  <node pkg="livox_mapping" type="pcl_load" name="pcl_load" output="screen"/>
  <node pkg="octomap_server" type="octomap_server_node" name="octomap_server"> 

     <!--栅格分辨率-->
    <param name="resolution" value="0.1" />
    
    <!--世界坐标系话题，一般为map-->
    <param name="frame_id" type="string" value="/livox_frame" />
 
    <param name="sensor_model/max_range" value="1000.0" />
    <param name="latch" value="true" />
 
     <!--截取的三维点云范围--> 
    <param name="pointcloud_max_z" value="100" />
    <param name="pointcloud_min_z" value="-100" />
 
    <param name="graound_filter_angle" value="3.14" />
    
     <!--订阅的点云消息名称 PointClout2，即上一步发布出来的点云topic--> 
     <!-- cloud_in是octomap默认的输入话题，做一下映射-->
    <remap from="/cloud_in" to="/pcl_output" />
    <remap from="/projected_map" to="/map" />
  </node>
 
</launch>
